docs/getting-started/quickstart.md

```diff
@@ -1,6 +1,23 @@
 # Quickstart
 
-In this tutorial we will
+## What is Feast?
+
+Feast (Feature Store) is an open-source feature store designed to facilitate the management and serving of machine learning features in a way that supports both batch and real-time applications.
```

Comment:
```suggestion
Feast (Feature Store) is an open-source feature store designed to facilitate the management and serving of machine learning features in a way that supports both batch and real-time applications.

* *For Data Scientists*: Feast is a a tool where you can easily define, store, and retrieve your features for both model development and model deployment. By using Feast, you can focus on what you do best: build features that power your AI/ML models and maximize the value of your data.

* *For MLOps Engineers*: Feast is a library that allows you to connect your existing infrastructure (e.g., online database, application server, microservice, analytical database, and orchestration tooling) that enables your Data Scientists to ship features for their models to production using a friendly SDK without having to be concerned with software engineering challenges that occur from serving real-time production systems. By using Feast, you can focus on maintaining a resilient system, instead of implementing features for Data Scientists.

* *For Data Engineers*: Feast is a centralized repository and single source of truth for feature data, which allows you to focus on integrating data sources to one location to minimize redundancy in data pipelines—for both offline analytics and for online serving. By focusing on integrating the data sources, you enable your Data Scientists to be able to create new features from those data sources without having to depend on you for every new change.
```

Reply:
"MLOps Engineers" has quite the run-on sentence. Also, just one perspective but I personally only thing of feast across 2 different user profiles: data scientists and engineers.

For engineers, I might say something like:

"Feast provides a centralized catalog for storing feature definitions allowing one to maintain a single source of truth for feature data. It provides the abstraction for reading and writing to many different types of offline and online data stores. Using either the provided python SDK or the feature server service, users can write data to the online and/or offline stores and then read that data out again in either low-latency online scenarios for model inference, or in batch scenarios for model training."

Reply:
Changed the section For Data Engineers to your suggestion. @franciscojavierarceo should we remove For MLOps Engineers section ? what do you say ?

Reply:
Let's keep it with the three personas outlined. I hear Matt's feedback and I think it's reasonable but we can revise in another PR because I would like to fix the run-on but I do want to keep the different roles outlined. An engineer can choose to do all three roles if they want, but it's helpful to know the hats being worn.